cmd/manager/main.go

```diff
@@ -76,16 +77,26 @@ func main() {
 		tlsCertPath             string
 		leaderElectionNamespace string
 		version                 bool
+		loglvl                  string
 	)
 	flag.StringVar(&clusterOperatorName, "clusterOperatorName", "", "configures the name of the OpenShift ClusterOperator that should reflect this operator's status, or the empty string to disable ClusterOperator updates")
 	flag.StringVar(&defaults.Dir, "defaultsDir", "", "configures the directory where the default CatalogSources are stored")
 	flag.BoolVar(&version, "version", false, "displays marketplace source commit info.")
 	flag.StringVar(&tlsKeyPath, "tls-key", "", "Path to use for private key (requires tls-cert)")
 	flag.StringVar(&tlsCertPath, "tls-cert", "", "Path to use for certificate (requires tls-key)")
 	flag.StringVar(&leaderElectionNamespace, "leader-namespace", "openshift-marketplace", "configures the namespace that will contain the leader election lock")
+	flag.StringVar(&loglvl, "verbosity", "info", "Sets verbosity level of logger with default verbosity info level. See https://github.com/sirupsen/logrus for other verbosity levels.")
```

Comment:
Could we change the argument to be `level` as `verbosity` is a different concept when it comes to logrus?

Reply:
Yep I like that.